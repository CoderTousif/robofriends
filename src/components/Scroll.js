import React from 'react';

const Scroll = (props) => {
    return (
        <div
            style={{
                overflowY: 'scroll',
                height: '70vh',
                position: 'sticky',
                top: '0',
                zIndex: '5',
                padding: '10px',
                border: '1px solid black',
                // background:
                //     'linear-gradient(to left, rgba(7,27,82,1) 0%, rgba(0,128,128,1) 100%)',
            }}
        >
            {props.children}
        </div>
    );
};

export default Scroll;
