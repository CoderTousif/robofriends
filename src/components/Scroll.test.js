import React from 'react';
import Scroll from './Scroll';
import { shallow } from 'enzyme';

it('should Scroll component', () => {
    expect(shallow(<Scroll />)).toMatchSnapshot();
});
