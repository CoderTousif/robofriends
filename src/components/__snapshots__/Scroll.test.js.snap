// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should Scroll component 1`] = `ShallowWrapper {}`;
